### ISSUE TITLE
Confirmation email template

### ISSUE THREAD

**Issue by yoavaviram** [your-digital-rights/yourdigitalrights.org#162]
The confirmation email should contain the following:
- Thank you message, and a reminding that we will email back in 30 days to check if the request was handled to the user's satisfaction.
- Link to request record page (postponed for now #163 )
- Request for donation
- Request to social share
- Request to install browser extension
- Recommendations of other companies to opt-out of (for example top 10)
- Link to delete all user data

All links in the email should include the following campaign tracking code: '?mtm_campaign=followup-emails&mtm_keyword=confirmation-email&mtm_source=email'

The DB should be updated that a confirmation email was sent.

**Comment by roughprada** [comment]
@yoavaviram are we creating 2 different templates for CCPA and GDPR? it will be good o briefly explain the process here
![confirmation email](https://user-images.githubusercontent.com/5957010/111900025-07da3d00-8a28-11eb-8a6c-de3873be3a79.png)

**Comment by roughprada** [comment]
Image copy:

- title: What happend next?
- submit your erasure request
- If you haven't heard back from the organization after 30 days, we will send you a reminder
- After 30 days you can escalate to the local Data Protection Authority

Email: 

Thank you for using yourdigitalrights.org,
An organization has one month to comply with a request. The deadline can be extended to 2 additional months taking into account the complexity and number of requests. In any case, the organization must inform you of such extension within one month from the receipt of the request.

After 30 days, we will send you a reminder so you can follow up on the request.

If you find our service useful and you’d like to support us, follow this link to our donation page.

Yourdigitalrights.org team

Footer:

Why not opting out from Data Brokers?
Data Brokers are companies which collect and sell personal data, typically without your knowledge or consent. These are some of the top data brokers

Share: 

Share with facebook, twitter or LinkedIn
Download our browser extension

**Comment by yoavaviram** [comment]
I think the donation buttons should appear on the email, not redirect to a page.

**Comment by yoavaviram** [comment]
This email looks kind of corporate to me. Can we make it more friendly? Also would be great to have an email template for all our emails.

**Comment by roughprada** [comment]
@yoavaviram I agree. I think the personality can come from the copy and the top banner, the rest can be just easy and plane, don't you think? if you can have a look at the copy, I'd think about a nice illustration to go on the top  banner

**Comment by yoavaviram** [comment]
+1

**Comment by roughprada** [comment]
@yoavaviram I think we can take this as a template:

- Top Banner
- Greetings
- Copy
- Data Brokers
- Share
- Unsubscribe

The banner and copy can change with the type of content. Can you look at the copy?
Also, we will need to have a "successfully unsubscribed" screen. I am assuming simple as a thank you page?

**Comment by roughprada** [comment]
@yoavaviram just cracked something quick to ad the copy on the unsubscribe section: https://zpl.io/VDj87je

**Comment by roughprada** [comment]
@yoavaviram just cracked something quick to ad the copy on the unsubscribe section: https://zpl.io/VDj87je

**Comment by honzie** [comment]
Working on the email templates (HTML and TXT) here: https://github.com/your-digital-rights/aws-backend/blob/feat/confirmation-email/outbound-mail/confirmation-template.html

**Comment by roughprada** [comment]
@honzie @yoavaviram  We will be going without Banner to not to block you on this.
Designs can be found here: https://zpl.io/bWM5B6p

**Comment by yoavaviram** [comment]
Ok

**Comment by yoavaviram** [comment]
@honzie I happened to stumble on this react lib, is it worth considering? https://www.npmjs.com/package/mjml-react

**Comment by yoavaviram** [comment]
Copy: https://docs.google.com/document/d/1vCek72GWZUkLRWmQ1Raxb5h7bCvWtUCfCS2F4S7G3ZM/edit#heading=h.6cdl49dvfqpp

**Comment by honzie** [comment]
Here's a PR that includes the email templates: your-digital-rights/aws-backend#7

**Comment by honzie** [comment]
@yoavaviram : I've requested access to the Google doc.

**Comment by honzie** [comment]
https://github.com/your-digital-rights/aws-backend/pull/8
